core/src/main/resources/jenkins/security/apitoken/ApiTokenPropertyEnabledNewLegacyAdministrativeMonitor/message.properties

```diff
@@ -0,0 +1,2 @@
+warningMessage=Users without Legacy API Token have the capability to create new Legacy API Token. <br />\
```

Comment:
nit: I would reword this to "Users without legacy API tokens are able to generate a new legacy token.\<br/\>This behavior is supported for backwards compatibility, but legacy API tokens are deprecated and are not recommend for long-term use. Users should instead generate API tokens as needed from the user configuration page."

Reply:
correction applied with fixed typo: recommend**ed**